dev/install-common-deps.sh

```diff
@@ -75,7 +75,6 @@ pip install --no-dependencies tests/resources/mlflow-test-plugin
 
 # Print current environment info
 pip install aiohttp
-python dev/show_package_release_dates.py
```

Comment:
`python dev/show_package_release_dates.py` is run here, but we need to scroll a lot to reach the table listing package versions.